flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/drivers/parameter/LongParameter.java

```diff
@@ -52,16 +52,6 @@ public LongParameter(ParameterizedBase owner, String name) {
 	public LongParameter setDefaultValue(long defaultValue) {
 		super.setDefaultValue(defaultValue);
 
-		if (hasMinimumValue) {
-			Util.checkParameter(defaultValue >= minimumValue,
-				"Default value (" + defaultValue + ") must be greater than or equal to minimum (" + minimumValue + ")");
-		}
-
-		if (hasMaximumValue) {
-			Util.checkParameter(defaultValue <= maximumValue,
-				"Default value (" + defaultValue + ") must be less than or equal to maximum (" + maximumValue + ")");
-		}
-
```

Comment:
I'm no expert here, but can you elaborate on why you removed these checks for the default value being in line with the min and max?

I can only guess that this must be that way in case where the default parallelism is not within the bounds of the configured one. I'm wondering whether the `LongParameter` is used somewhere else were these bounds are expected to hold though.